pages/over-ons.tsx

```diff
@@ -1,11 +1,75 @@
-import { Container, Layout } from '@components'
+import React from 'react'
+
+import { Box, Heading, Stack, Text } from '@chakra-ui/react'
+import { GetStaticProps } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+import { Container, Hero, Layout } from '@components'
 
 const OverOns = (): JSX.Element => {
+  const { t } = useTranslation()
   return (
     <Layout>
-      <Container>Over Ons</Container>
+      <Container>
+        <Hero
+          isFullHeight={false}
+          title={t('about-us.title')}
+          // image={AboutImage}
```

Comment:
You can try referring an image path inside public folder. For instance, let say we have `public/images/about-us.jpg`. You can simply provide as below:

```suggestion
         image={`${process.env.NEXT_PUBLIC_SITE_URL}/images/about-us.jpg`}
```


### Description

Actually image paths are being handled by `getImageUrl` function. This means you can provide the image path as shown in the following examples:

1. Provide an image object coming from strapi backend _`image={image as IUploadFile}`_
2. Provide a relative image path, (eg: /uploads/image.png). _(In this case we expect that you provide a relative url from strapi backend)_ `image={image.url}`
3. Provide an absolute image path from any source. (eg: unsplash).

In this case we provide an absolute path from our own frontend url.


 ```jsx
export const getImageUrl: GetImageUrlType = (image, type?) => {
  const basePath = process.env.NEXT_PUBLIC_ADMIN_URL
  if (image == null) return ''

  if (typeof image === 'string')
    return image.startsWith('http') ? image : `${basePath}${image}`

  const imagePath =
    (type && image.formats && image.formats[type].url) || image.url

  return `${basePath}${imagePath}`
}
```